lib/errorSerialization.js

```diff
@@ -0,0 +1,18 @@
+var serializeError = require('serializerr');
+var deserializeError = require('deserialize-error');
```

Comment:
I don't see a reason for pulling in two separate packages (along with their dependencies) merely to serialize an error object; we know which properties we need serialized, and they were taken care of in the previous serializer implementation. However, the bug relied in the error not being deserialized before being passed to webpack, which you've fixed.

We can remove the dependencies and build a deserializer routine that yields an Error object with the necessary properties.